operations/helm/policies/policies_test.rego

```diff
@@ -13,17 +13,17 @@ some_violaions if {
 }
 
 test_no_namespace_not_allowed if {
-	some_violaions with input as {"metadata": {"name": "resource"}}
+	some_violaions with input as [{"contents": {"metadata": {"name": "resource"}}}]
```

Comment:
copy-paste error: definition and usage missing t everywhere :  `some_violaions` -> `some_violations`